app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainFragment.kt

```diff
@@ -41,11 +43,11 @@ class MainFragment : Fragment() {
          * Declare field and other necessary variables
          */
         val view = inflater.inflate(R.layout.main_fragment, container, false)
-        val recoveredTxt: TextView = view.findViewById<TextView>(R.id.recovered_num) as TextView
-        val confirmedTxt: TextView = view.findViewById<TextView>(R.id.confirmed_num) as TextView
-        val deathsTxt: TextView = view.findViewById<TextView>(R.id.deaths_num) as TextView
-        val countryBarChart: BarChart = view.findViewById<BarChart>(R.id.by_country_bar_graph) as BarChart
-        val countryListView: ListView = view.findViewById<ListView>(R.id.country_list_view) as ListView
+        val recoveredTxt: TextView = view.findViewById(R.id.recovered_num) as TextView
+        val confirmedTxt: TextView = view.findViewById(R.id.confirmed_num) as TextView
+        val deathsTxt: TextView = view.findViewById(R.id.deaths_num) as TextView
+        val countryBarChart: BarChart = view.findViewById(R.id.by_country_bar_graph) as BarChart
+        val countryListView: ListView = view.findViewById(R.id.country_list_view) as ListView
```

Comment:
No need for the <type> since the "as (type)" is used: https://kotlinlang.org/docs/reference/coding-conventions.html#avoiding-redundant-constructs